packages/web-components/src/components/va-accordion/va-accordion.e2e.ts

```diff
@@ -14,7 +14,7 @@ describe('va-accordion', () => {
       <va-accordion class="hydrated">
         <mock:shadow-root>
           <div class="usa-accordion">
-            <button aria-controls="" aria-expanded="false" aria-label="expand-all-aria-label" class="va-accordion__button">
+            <button aria-label="expand-all-aria-label" class="va-accordion__button">
```

Comment:
/cc @1Copenut 

TODO: Add a `data-cytest="NAME"` attribute here and update the Cypress `expandAccordions()` helper. Could add an optional argument to accept a string value, but definitely need to ensure the helper doesn't break or tests start failing.